sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/json/JSONOptions.scala

```diff
@@ -54,31 +54,31 @@ private[sql] class JSONOptions(
   val samplingRatio =
     parameters.get("samplingRatio").map(_.toDouble).getOrElse(1.0)
   val primitivesAsString =
-    parameters.get("primitivesAsString").map(_.toBoolean).getOrElse(false)
+    parameters.get("primitivesAsString").exists(_.toBoolean)
```

Comment:
I think this case actually map and getOrElse is easier to read ..

Reply:
I'm OK with that too. It's more performant if it were a list, but slightly less readable, and this is an Option

Reply:
It's worth sacrificing performance for readability. If we agree that `exists/forall` is not easier to read, I will close this PR and keep it as it is.

Reply:
Yeah I think we might just leave it as is

Reply:
OK. I close this PR. Thanks! @srowen @HyukjinKwon